blocks/HARDWARE/PROTOCOLS/GPIB/PROLOGIX/PROLOGIX_EOI/PROLOGIX_EOI.py

```diff
@@ -0,0 +1,64 @@
+import serial
+from flojoy import flojoy, SerialConnection, DataContainer, String
+from typing import cast, Optional, Literal
+
+
+@flojoy(inject_connection=True)
+def PROLOGIX_EOI(
+    connection: SerialConnection,
+    default: Optional[DataContainer] = None,
+    eoi: bool = True,
+    eos: Literal["CR+LF", "CR", "LF", "None"] = "None",
+) -> String:
+    """Sets the EOI and EOS settings for the Prologix GPIB-USB adapter.
```

Comment:
How can someone know which EOI and IOS to use? Is it easy to know? Or is it worth mentioning how to find the info?

Reply:
Honestly, I'm not sure. It might be instrument dependent. At first I thought it was necessary, but for me it didn't seem to make a difference.

Reply:
You have one in the office there if you want to try it out with an instrument

Reply:
Oh, I'll try it right now